modules/apps/27-interchain-accounts/controller/client/cli/cli.go

```diff
@@ -1,6 +1,7 @@
 package cli
 
 import (
+	"github.com/cosmos/cosmos-sdk/client"
```

Comment:
Nit: this import should go second and separated by empty line from the cobra import.